proposals/0216-widget-support.md

```diff
@@ -522,62 +587,157 @@ This functional group should be provided to the app developers as a conditional
 1. Changes to RPC service as new RPCs and parameters are introduced
-2. Changes to State Controller to allow HMI level transitions for widgets and screens
-3. Changes to Request Controller as RPCs from one app can be addressed to different screens
+2. Changes to State Controller to allow HMI level transitions for widgets and windows
+3. Changes to Request Controller as RPCs from one app can be addressed to different windows
 
-T screen managers should be refactored to read screen capabilities notifications as well as the deprecated parameters.
+The window managers should be refactored to read window capabilities notifications as well as the deprecated parameters.
+
+## Additional features
+
+This section describes a possible feature to allow widgets per app or service type. Instead of providing only app specific widgets, an infotainment systems with widget support can also offer widgets per type. As an example The infotainment system can offer a "Media Player" widget which will present the widget of the active/audible media application. This can include media sources offered by the infotainment system such as Radio, Bluetooth or USB playback. A similar user experience can be offered to other app types like navigation, weather, (voice) communication or telephony, voice assitant etc.
+
+There are different approaches to provide this feature.
+
+### App HMI type
+
+```xml
+<function name="CreateWindow" messagetype="request">
+  :
+  <param name="widgetType" type="AppHMIType" mandatory="false">
+    <description>
+      Allows an app to create a widget related to a specific HMI type.
+      As an example if a `MEDIA` app becomes active, this app becomes audible and is allowed to play audio. Actions such as skip or play/pause will be
+      directed to this active media app. In case of widgets, the system can provide a single "media" widget will acts as a placeholder for the active media app.
+
+      It is only allowed to have one window per HMI type. This means that a media app can only have a a single MEDIA widget. Still the app can create widgets omitting this parameter or with specifying the HMI type `DEFAULT`. Those widgets would be available to the user independent of the HMI type.
+
+      This parameter is related to widgets only. The default main window, which is pre-created during app registration, will be created based on the HMI types specified in the app registration request.
+    </descripion>
+  </param>
+</function>
+```
+
+Allows an app to create a widget related to a specific HMI type.
+As an example if a `MEDIA` app becomes active, this app becomes audible and is allowed to play audio. Actions such as skip or play/pause will be
+directed to this active media app. In case of widgets, the system can provide a single "media" widget will acts as a placeholder for the active media app.
```

Comment:
```suggestion
directed to this active media app. In case of widgets, the system can provide a single "media" widget which will act as a placeholder for the active media app.
```